data/db/mil-olympus.xml

```diff
@@ -203,7 +203,16 @@
         <mount>Micro 4/3 System</mount>
         <cropfactor>2</cropfactor>
     </camera>
-
+    
+    <camera>
+        <maker>Olympus Corporation</maker>
+        <maker lang="en">Olympus</maker>
+        <model>E-M10MarkIV</model>
```

Comment:
I see that the names in this file are not consistent with respect to spaces. Can I just check that this is working for you (that is, automatically selecting the right thing from exif data) in some program?